index.ts

```diff
@@ -209,3 +212,55 @@ cron.onEvent("partition-registrar", new CallbackFunction('event-gen-callback', {
         })
     }
 }));
+
+const eventGenRole = new aws.iam.Role("eventGenLambdaRole", {
+    assumeRolePolicy: JSON.stringify(lambdaAssumeRolePolicy),
+});
+
+const eventGenLambdaAccess = new aws.iam.RolePolicyAttachment("event-gen-lambda-access", {
+    role: eventGenRole,
+    policyArn: aws.iam.ManagedPolicies.AWSLambdaFullAccess,
+});
+
+const eventGenKinesisAccess = new aws.iam.RolePolicyAttachment("event-gen-kinesis-access", {
+    role: eventGenRole,
+    policyArn: aws.iam.ManagedPolicies.AmazonKinesisFullAccess,
+});
+
+
+const eventCron = new aws.cloudwatch.EventRule("event-gen-cron", {
+    scheduleExpression: "rate(1 minute)",
+});
+
+cron.onEvent("event-generator", new CallbackFunction('event-gen-callback', {
```

Comment:
Using the wrong cron, change to eventCron

Reply:
Hah, good catch! This would've manifested in very confusing ways.